tests/sync/test_syncing.py

```diff
@@ -7,25 +7,25 @@
 from kitovu.sync.plugin import smb
 
 
-def test_find_plugin_builtin():
-    plugin = syncing._find_plugin('smb')
-    assert isinstance(plugin, smb.SmbPlugin)
+class TestFindPlugin:
```

Comment:
The diff is kinda hard to read, but it looks like you only added this class, right? Why?

Reply:
Yes, I've added the class.

I prefer to have tests grouped by what they do. This can be done either by one file per test group or by grouping them within a file.

I've prepared it so when I add tests for other methods in this module that they are separated.